community/hazelcast-jet-enterprise/values-metadata.yaml

```diff
@@ -0,0 +1,638 @@
+      description: "How many probe attempts Kubernetes makes before giving up"
+      type: "number"
+      immutable: false
+      required: false
+
+resources:
+  __metadata:
+    label: "Resources"
+    description: "Resource requests and limits configuration"
+  requests:
+    cpu: 
+      __metadata: 
+        label: "CPU request"
+        description: "The requested CPU"
+        type: "string"
+        required: false   
+    memory: 
+      __metadata: 
+        label: "Memory request"
+        description: "The memory request"
+        type: "string"
+        required: false   
+  limits:
+    cpu: 
+      __metadata: 
+        label: "CPU limit"
+        description: "The CPU limit"
+        type: "string"
+        required: false   
+    memory: 
+      __metadata: 
+        label: "Memory limit"
+        description: "The memeory limit"
+        type: "string"
+        required: false   
+
+service:
+  __metadata:
+    label: "Service"
+    description: "Service configuration"
+  type:
+    __metadata:
+      name: "Type"
+      label: "Service Type"
+      description: "Kubernetes allow you to specify what kind of service you want"
+      type: "string"
+      immutable: false
+      required: true
+      options:
+        - label: "Cluster IP"
+          value: "ClusterIP"
+        - label: "Node Port"
+          value: "NodePort"
+        - label: "Load Balancer"
+          value: "LoadBalancer"
+  port:
+    __metadata:
+      name: "port"
+      label: "Port"
+      description: "Port number"
+      type: "number"
+      immutable: false
+      required: true
+  clusterIP:
+    __metadata:
+      name: "clusterIP"
+      label: "ClusterIP"
+      description: "Cluster IP set to None makes the service headless. It is required if DNS Lookup is used (https://github.com/hazelcast/hazelcast-kubernetes#dns-lookup)"
+
+
+rbac:
+  __metadata:
+    label: "RBAC"
+    description: "RBAC roles and bindings"
+  create:
+    __metadata:
+      name: "create"
+      label: "Create"
+      description: "Create the RBAC roles and bindings required for Hazelcast Jet cluster auto discovery via Kubernetes api"
+      type: "boolean"
+      immutable: false
+      required: true
+
+serviceAccount:
+  __metadata:
+    label: "Service Account"
+    description: "Service Account configuration"
+  create:
+    __metadata:
+      name: "create"
+      label: "Create"
+      description: "Create the Service Account required for Hazelcast Jet cluster auto discovery via Kubernetes api"
+      type: "boolean"
+      immutable: false
+      required: true
+  name:
+    __metadata:
+      name: "name"
+      label: "Name"
+      description: "Name of the service account; if not set, a name is generated using the fullname template"
+      type: "string"
+      immutable: false
+      required: false
+
+securityContext:
+  __metadata:
+    label: "Security Context Configuration"
+    description: "Security Context Configuration"
```

Comment:
Please update this description as well to be different and more helpful than the label.